.env.docker.example

```
@@ -0,0 +1,26 @@
+FRONTEND_URL=http://localhost
+BACKEND_URL=http://localhost
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BACKEND_URL key should go before the FRONTEND_URL key

(UnorderedKey)